moveit_core/macros/include/moveit/macros/declare_ptr.h

```diff
@@ -0,0 +1,55 @@
+/*********************************************************************
```

Comment:
``class_forward`` has always been somewhat miss-named by also declaring shared_ptrs. I don't mind too much either way how we do it, just seemed slightly more complicated your way.